Dockerfile.ocp

```diff
@@ -1,4 +1,4 @@
-ARG BUILDER_IMAGE=brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.23
+ARG BUILDER_IMAGE=brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.24
```

Comment:
This image is not ready yet. Konflux will fail. But it is a matter of time to have it. So, we can safely update it too.